background/services/internal-ethereum-provider/index.ts

```diff
@@ -384,14 +384,15 @@ export default class InternalEthereumProviderService extends BaseService<Events>
   async getCurrentOrDefaultNetworkForOrigin(
     origin: string
   ): Promise<EVMNetwork> {
-    const currentNetwork = await this.db.getCurrentNetworkForOrigin(origin)
+    /*const currentNetwork = await this.db.getCurrentNetworkForOrigin(origin)
```

Comment:
Was there any reason that ethereum used this function? Does that reason no longer apply to us?

Reply:
Currently the db returns the wrong network in getCurrentNetworkForOrigin and I haven't fixed it yet, so I commented the code out. I've created an issue for this: https://github.com/PelagusWallet/pelagus-extension/issues/14